wolftpm/tpm2.h

```diff
@@ -1915,8 +1915,10 @@ WOLFTPM_API TPM_RC TPM2_GetCapability(GetCapability_In* in,
 
 typedef struct {
     TPMI_YES_NO fullTest;
+    TPMI_YES_NO continueTest;
```

Comment:
This is not part of the specification. Can you please share a link to the U-Boot code that does this? Perhaps this was part of TPM 1.2? The TPM 2.0 specification is here: https://trustedcomputinggroup.org/resource/tpm-library-specification/